Project.toml

```diff
@@ -34,7 +34,7 @@ Distributions = "0.21,0.22,0.23, 0.24, 0.25"
 MLJBalancing = "0.1"
 MLJBase = "1"
 MLJEnsembles = "0.4"
-MLJFlow = "0.4"
+MLJFlow = "0.4.2"
```

Comment:
Shouldn't this be 0.5 (or 0.5.1)?

Reply:
No, the branch name was a confusion with MLFlowClient.jl last version. Here we are upgrading MlFlow.

Reply:
Oh right. Thanks for the clarification.

The change to use `apiroot` is a technically breaking change though, correct?

It won't trigger a breaking release to MLJ, as we document MLJFlow integration as experimental, but I definitely want to mention any breaking behaviour in our release notes. Can you please summarise any breaking behaviour below so I can report it?

Reply:
This change, outside the experimental context, contains a breaking change in the usability side (dependency safe).
It introduces the user capability to define its own API route, removing any side logic referencing specific platforms (e.g. DagsHub). It means that we now have a method agnostic way to know if `api`, `ajax-api`, or any other one is being used.